eth/v1/node.pb.go

```diff
@@ -18,6 +14,9 @@ import (
 	grpc "google.golang.org/grpc"
 	codes "google.golang.org/grpc/codes"
 	status "google.golang.org/grpc/status"
+	io "io"
+	math "math"
+	math_bits "math/bits"
```

Comment:
Do you think `go imports` will fix back this update?

Reply:
Terence has run `go imports` in https://github.com/prysmaticlabs/ethereumapis/pull/209 and wanted to make sure we do not change something that will require separate PR to fix.